src/states/adult.rs

```diff
@@ -247,11 +249,18 @@ impl Adult {
             .gen_pfx_info
             .latest_info
             .member_nodes()
+            .filter(|node| node.public_id() != self.id())
             .map(P2pNode::connection_info)
             .cloned()
             .collect_vec();
```

Comment:
Yes, I think we need to be careful with poke, I think if we do that we'll also want to ensure Demoted Elder can answer the Poke. Alternative way to improve this would be to limit the Adults that poke, could be ELDER_SIZE * 2 oldest set that do the poke, maybe we give an age limit in GenesisUpdate/NodeApproval under which poke does not happen.

I think this require some consideration, will raise an issue first.

Reply:
Sounds good. :+1: for raising an issue.

Reply:
Raised issue https://github.com/maidsafe/routing/issues/1952